.trivyignore

```diff
@@ -4,4 +4,18 @@ CVE-2024-24790
 CVE-2023-49569
 CVE-2024-3817
 CVE-2024-23652
-CVE-2024-23653
\ No newline at end of file
+CVE-2024-23653
```

Comment:
I would say no harm to upgrade golang version, not sure if there is any breaking change for that

Reply:
+1 @NoSugarCoffee - lets please try updating Golang version first. If that fails lets ignore the vulnerability temporarily